src/utils/viewer/viewer.cpp

```diff
@@ -198,6 +197,66 @@ Key Commands:
     }
   }
 
+  /**
+   * @brief vector holding past agent locations to build trajectory
+   * visualization
+   */
+  std::vector<Magnum::Vector3> agentLocs_{
```

Comment:
I assume this will be moved to a test file before merging.

Reply:
Not really - i want to enable the viewer to be used to record a trajectory.